docs/participate/devnet.md

```diff
@@ -1,10 +1,51 @@
 ---
-sidebar_position: 1
+sidebar_position: 1 
 ---
 # Devnet
 
-:::tip Devnet coming soon!
+:::tip Devnet Program is live!
 
-You can signup [here](https://airtable.com/appr3wKRsn4VoOvpi/pagxxT6QcL0OwGH0e/form), and we'll reach out to you once it goes live! 
+You can mint our Devnet slot from our [Mint page](https://devnet-mint.powerloom.dev/). Once minted, you can navigate to the [getting started](../build-with-powerloom/devnet/getting-started.md) section to kickstart your devnet journey.  
 
 :::
+
+## What is Powerloom Devnet? 
+
+Powerloom Devnet is a special development environment that allows developers to easily build and test data-driven applications on Powerloom. It provides a sandbox where you can experiment, learn, and create without impacting the main network.
+
+Devnet enables you to efficiently customize and analyze blockchain data to power your applications. Whether you're building a simple prototype or a complex applications, Devnet makes the development process smooth and straightforward.
+
+## Why Use Powerloom Devnet?
+
+There are several key benefits to using Powerloom Devnet for your development needs:
+
+- **Easy Onboarding**: Minting a Devnet slot is quick and simple through our user-friendly minting page. You can get started building right away without any lengthy setup process.
+
+- **Comprehensive Datasets**: Powerloom provides access to rich and diverse data markets. This allows you to work with real-world data and test your applications under realistic conditions. 
+
+- **Collaborative Environment**: Devnet fosters a vibrant developer community where you can connect with other builders, share knowledge, and collaborate on projects. Leverage the collective expertise to accelerate your development efforts.
+
+## What Can You Build with Devnet?
+
+The possibilities are truly endless when it comes to building data applications on Powerloom Devnet. Here are a few exciting examples:
+
+1. **Data Dashboards**: Create interactive visualizations and dashboards that provide insights into blockchain metrics, token economies, user behavior, and more. Empower users with data-driven decision making.
```

Comment:
Let's only write things that users can build today. We can expand later. 
Better to keep it simple and to the point IMO.